spec.v2.yaml

```diff
@@ -357,3 +357,24 @@ data:
             - xxx-0:1-1.module_deadbeef.x86_64
             - xxx-0:1-1.module_deadbeef.i686
             - xyz-0:1-1.module_deadbeef.x86_64
+        # The rpm-map exists to link checksums from repomd to specific
+        # artifacts produced by this module. Any item in this list must match
+        # an entry in the data.artifacts.rpms section.
+        # Optional.
+        rpm-map:
+          # The digest-type of this checksum. Required.
+          sha256:
+            # The checksum of the artifact being sought. Required.
+            ee47083ed80146eb2c84e9a94d0836393912185dcda62b9d93ee0c2ea5dc795b:
+              # The RPM name. Required.
+              name: bar
+              # The RPM epoch. Optional
+              epoch: 0
```

Comment:
I don't know if we should make this property optional. It's not optional in rpm-md, as null epoch is automatically forced to zero epoch.

Reply:
I made it optional for reading, but libmodulemd outputs it unconditionally. Is that good enough or do you feel strongly that it should be mandatory in all cases?

Reply:
I think it should remain mandatory in all cases. Remember, this should be consistent for other YAML-based parsers for modulemd.